public/css/main.css

```diff
@@ -0,0 +1,7 @@
+/* Links style reset */
+body a {
+  display: contents;
```

Comment:
The declarations should be alphabetized.

Reply:
Done